PDFWriter/CMakeLists.txt

```diff
@@ -405,7 +405,7 @@ if(PDFHUMMUS_NO_TIFF)
     target_compile_definitions(PDFWriter PUBLIC PDFHUMMUS_NO_TIFF=1)    
 else(PDFHUMMUS_NO_TIFF)
     if(USING_UNBUNDLED_TIFF)
-        target_link_libraries(PDFWriter TIFF::TIFF)
+        target_link_libraries(PDFWriter ${TIFF_LIBRARIES})
```

Comment:
why is this change required? i'd rather use targets.

Reply:
maybe even just add it in your own cmakefiles.

Reply:
yeah, i reckon you should be linking to the target directly. And i recommend using target_link_libraries(YOUR_LIBRARY TIFF:CXX) in addition to  target_link_libraries(YOUR_LIBRARY TIFF:TIFF) to get both the tiff target and C++ wrapper.

It does still mean that find_package should bring in the CXX component.
not sure if you can do this by just adding find_package(TIFF COMPONETS CXX) in your cmake file. im not sure how multiple find_package work...but it might work. if not, try adding CXX as you did to the pdfwriter as you did (though maybe without the "required" bit).

in any case, i'll be working on an upgrade to the bundled libtiff using the source code changes you made, so at least this part is a bit more compatible to the new version.

Reply:
given that PDFWriter itself does not use the C++ wrapper, there's no reason for it to look or link against it, unless there's no other way to overcome this, one was suggested here.